src/main/resources/assets/computercraft/lua/rom/programs/http/webrun

```diff
@@ -0,0 +1,25 @@
+--Made by Wilma456
+local args = {...}
+
+if not args[1] then
+  print("Usage: webrun <url> <args>")
+  return
+end
+
+if not http then
+  print("Please enable http in the config")
+  return
+end
+
+local test,text = http.checkURL(args[1])
+if test == false then
```

Comment:
Not super important, but you can just do `if not test then...` instead of `if test == false then...`